### DISCUSSION TITLE
does stripe using markdoc for their api documentation https://docs.stripe.com/api/expanding_objects.

### DISCUSSION THREAD

**Discussion by the-vineethav** [markdoc/markdoc#519] [Q&A]
I also saw an old post related to this which is posted in 2022. https://github.com/markdoc/markdoc/discussions/50. I wanted to know current status. Any help is greatly appreciated.

ALso I couldnt find any detailed dcumentation on how to use markdoc for api documentation. Please direct me to the appropriate links

**Comment by varughese** [comment]
See https://github.com/markdoc/markdoc/discussions/449#discussioncomment-9874250
